lib/login/view/login_page.dart

```diff
@@ -14,18 +19,49 @@ class _LoginPageState extends State<LoginPage> {
   late String emailValue;
   late String passwordValue;
   bool passwordInvisible = false;
+  late LoginBloc bloc;
+
+  @override
+  void initState() {
+    super.initState();
+    bloc = LoginBloc();
+  }
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        centerTitle: true,
-        automaticallyImplyLeading: false,
-        backgroundColor: AppColors.primaryColor,
-        title: Text(AppLocalizations.of(context)!.loginTitle),
-      ),
-      body: _buildBody(),
-    );
+    return BlocConsumer(
+        bloc: bloc,
+        builder: (context, Object? state) {
+          return Scaffold(
+            appBar: AppBar(
+              centerTitle: true,
+              automaticallyImplyLeading: false,
+              backgroundColor: AppColors.primaryColor,
+              title: Text(AppLocalizations.of(context)!.loginTitle),
+            ),
+            body: _buildBody(),
+          );
+        },
+        listener: (context, state) {
+          if (state is SuccessfulLogin) {
+            Navigator.pushReplacement(
```

Comment:
we should use named route

Reply:
named routes will be added in another pull request